sources/Inputs/src/NetworkManager.cpp

```diff
@@ -86,6 +91,7 @@ NetworkManager::buildTree() {
     if (bus1 && bus2) {
       nodes_[bus1->getID()]->neighbours.push_back(nodes_[bus2->getID()]);
       nodes_[bus2->getID()]->neighbours.push_back(nodes_[bus1->getID()]);
+      LOG(debug) << "Node " << bus1->getID() << " connected to " << bus2->getID() << LOG_ENDL;
```

Comment:
maybe we could add the name of the line/2wt/3wt in the message?

Reply:
OK